ci-operator/step-registry/redhat-chaos/pvc-scenario/redhat-chaos-pvc-scenario-workflow.yaml

```diff
@@ -0,0 +1,19 @@
+workflow:
+  as: redhat-chaos-pvc-scenario
+  steps:
+    test:
+      - ref: redhat-chaos-pvc-scenario
+    env:
+      TARGET_NAMESPACE: "kraken"
+      PVC_NAME: "kraken-test-pvc"
+      POD_NAME: "kraken-test-pod"
```

Comment:
Maybe we can target Prometheus pod instead of test pod to understand the impact when it fills up? Thoughts?